model/src/main/proto/oppia_logger.proto

```diff
@@ -80,6 +110,105 @@ message EventLog {
     int32 sub_topic_id = 2;
   }
 
+  // Structure of a start card context.
```

Comment:
Perhaps something like: ``Represents the event context for opening a card within an exploration.`` would be clearer context. Ditto for the other new context messages.

Reply:
Collapsed this context to support both card start and end related events. Modified documentation accordingly, PTAL.

Reply:
Much clearer--thanks!